underscore.js

```diff
@@ -558,6 +558,8 @@
 
   // Produce a duplicate-free version of the array. If the array has already
   // been sorted, you have the option of using a faster algorithm.
+  // The faster algorithm will not work with an iteratee if the iteratee is not a one-to-one function, so providing an iteratee will disable the faster algorithm
```

Comment:
Comments should be complete sentences, with punctuation, wrapped to an ~80 line width.